models/place.json

```diff
@@ -272,5 +160,107 @@
         "type": "Available"
       }
     }
+  ],
+  "properties": {
+    "@id": {
+      "$ref": "./place-@id.json"
+    },
+    "mainLanguage": {
+      "$ref": "./place-mainLanguage.json"
+    },
+    "name": {
+      "$ref": "./place-name.json"
+    },
+    "terms": {
+      "$ref": "./place-terms.json"
+    },
+    "calendarType": {
+      "$ref": "./place-calendarType.json"
+    },
+    "address": {
+      "$ref": "./place-address.json"
+    },
+    "geo": {
+      "$ref": "./place-geo.json"
+    },
+    "startDate": {
+      "$ref": "./place-startDate.json"
+    },
+    "endDate": {
+      "$ref": "./place-endDate.json"
+    },
+    "status": {
+      "$ref": "./place-status.json"
+    },
+    "openingHours": {
+      "$ref": "./place-openingHours.json"
+    },
+    "availableFrom": {
+      "$ref": "./place-availableFrom.json"
+    },
+    "availableTo": {
+      "$ref": "./place-availableTo.json"
+    },
+    "sameAs": {
+      "$ref": "./place-sameAs.json"
+    },
+    "created": {
+      "$ref": "./place-created.json"
+    },
+    "modified": {
+      "$ref": "./place-modified.json"
+    },
+    "creator": {
+      "$ref": "./place-creator.json"
+    },
+    "workflowStatus": {
+      "$ref": "./place-workflowStatus.json"
+    },
+    "languages": {
+      "$ref": "./place-languages.json"
+    },
+    "completedLanguages": {
+      "$ref": "./place-completedLanguages.json"
+    },
+    "typicalAgeRange": {
+      "$ref": "./place-typicalAgeRange.json"
+    },
+    "description": {
+      "$ref": "./place-description.json"
+    },
+    "priceInfo": {
+      "$ref": "./place-priceInfo.json"
+    },
+    "contactPoint": {
+      "$ref": "./place-contactPoint.json"
+    },
+    "bookingInfo": {
+      "$ref": "./place-bookingInfo.json"
+    },
+    "mediaObject": {
+      "$ref": "./place-mediaObject.json"
+    },
+    "image": {
+      "$ref": "./place-image.json"
+    },
+    "videos": {
+      "$ref": "./place-videos.json"
+    },
+    "labels": {
+      "$ref": "./place-labels.json"
+    },
+    "hiddenLabels": {
+      "$ref": "./place-hiddenLabels.json"
+    },
+    "bookingAvailability": {
+      "$ref": "./place-bookingAvailability.json"
+    }
+  },
```

Comment:
I think the `organizer` property is missing. Interesting problem with that though. `@id` is not required on `organizer.json`, because it's not required when creating a new organizer and when updating you can specify the id in the URL and the `@id` will be set correctly automatically. But when you import a place and you want to link an organizer, the `organizer.@id` is of course required. Not sure how to fix that in the schema without creating two schema's for organizer. Maybe we could try making `@id` required on organizer, but set it to `readOnly: true`?

Reply:
As discussed we resolve it now by creating a `organizer-refence.json` that will be used inside the `place.json`. Inside the documentation for the read place endpoints a link will be provided to the full embedded organizer.